drivers/windows/NAMESPACE

```diff
@@ -7,3 +7,4 @@ S3method(format,dide_shares)
 S3method(print,dide_clusterload)
 S3method(print,password)
 S3method(print,windows_path)
+importFrom(utils,read.csv)
```

Comment:
Any reason not to use `utils::read.csv` on use?

Reply:
Fixed